src/components/style/coin.css

```diff
@@ -0,0 +1,82 @@
+.coinlist {
+  margin-bottom: 64px;
+  display: flex;
+  flex-direction: column;
+  padding: 10px;
+}
+
+.all-item {
+  display: grid;
+  grid-template-columns: 50% 50%;
+  height: 30%;
+  gap: 10px;
+}
+
+input {
+  padding-left: 10px;
+  width: 20%;
+  height: 30px;
+  border-radius: 4px;
+  border: none;
+  background-color: #6097fc;
+  color: #fff;
+  justify-items: center;
+}
+
+input::placeholder {
+  color: #fff;
+}
+
+.list-item {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  list-style-type: none;
+  background-color: #4369b2;
+  color: #fff;
+  padding-left: 5px;
+  flex-wrap: wrap;
+}
```

Comment:
- [ ] [REQUIRED] Please fix your style issues!
- You have made a mistake in setting the background color of the cells, please recheck the given Behance design and solve them!  

![image](https://user-images.githubusercontent.com/63915024/188718708-9bda135c-0dbe-4419-b82f-bf6743ed87f1.png)

Reply:
thank you for the feedback even though I haven't seen any requirement on the project description I have implemented it
![new one](https://user-images.githubusercontent.com/32224137/188728392-b2fa1765-9b06-42e3-9b92-104150481b03.png)